src/etl/adapter/yd_parser.py

```
@@ -0,0 +1,122 @@
+            zip_path = os.path.join(tmp_dir, "download.zip")
+            extract_path = os.path.join(tmp_dir, "extracted")
+
+            # 2. Получаем прямую ссылку на скачивание через API Яндекса
+            response = requests.get(f"{self.base_url}{public_key}")
+            download_url = response.json().get('href')
+
+            if not download_url:
+                raise Exception("Не удалось получить ссылку на скачивание с Яндекс.Диска")
+
+            # 3. Скачиваем ZIP
+            with requests.get(download_url, stream=True) as r:
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)